lighthouse/src/main.rs

```diff
@@ -483,6 +483,16 @@ fn run<E: EthSpec>(
         };
     }
 
+    let sse_logging = {
+        if let Some(bn_matches) = matches.subcommand_matches("beacon_node") {
+            bn_matches.is_present("http") || bn_matches.is_present("gui")
```

Comment:
```suggestion
            bn_matches.is_present("gui")
```

~~The `--staking` flag also turns on the HTTP API.~~

Alternatively, what are your thoughts on only enabling this when the `--gui` flag is present? It would be make the roll-out softer, at the moment all stakers will be running elements of the async logging system in v4.2.0. I think its OK as-is, just running the idea past you.

Reply:
Yeah I forgot the reason I did this. 

I think we enable this as minimally as we can. On first thought seems right to only enable on gui. I'll have a think and modify if needed.

Reply:
I just updated my suggestion to only enable it with the `--gui` flag. That should me you can enact all of the suggestions in my PR via the Github UI.